tailwind.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    './ui/**/*.{js,jsx,ts,tsx}',
+    './node_modules/@metamask/design-system-react/**/*.{cjs,mjs}',
```

Comment:
`@metamask/design-system-react` is not added in this PR but will be how Tailwind ensures that the classnames that are needed will be generated